docs/docs/how-to/run-shards.md

```diff
@@ -0,0 +1,107 @@
+---
+authors: Fragcolor & contributors
+license: CC-BY-SA-4.0
+---
+
+# Run Shards
+
+!!! note
+    This guide is for new users unfamiliar with Visual Studio Code and the creation of new code files. 
+    
+    Click [here](#overview) to skip the tutorial and jump to the overview.
+
+How-to run Shards.
+
+Learn how to employ Shards to execute code and bring your works to fruition!
+
+Ensure that you have [Shards built](../how-to/build-shards.md) before attempting to run it.
+
+## Preparing the Script ##
+
+In VS Code, select “File → Open Folder…” (Ctrl+K+O) and select the Shards repository.
+
+Select the New Folder button ![New Folder Icon](assets/vscode-new-folder-icon.png) near the top of your Explorer window and name the folder “scripts”.
+
+![Create a new “scripts” folder.](assets/vscode-new-folder.png)
+
+Right-click the “scripts” folder and select “New File…”. Name the file “helloworld.edn”.
+
+![Create the “helloworld.edn” file in the “scripts” folder.](assets/vscode-new-file.png)
+
+??? ".edn"
+	EDN is a file format used by Clojure, one of the programming languages employed by Shards.
```

Comment:
Would it be alright to rephrase it to "EDN is a file format used by Clojure, a programming language Shards references from"?

If not, I can remove this segment about ".edn" files.

Reply:
I would remove any mention to Clojure for now to avoid confusion. Or maybe something like: "EDN is a file format used by other programming language such as Clojure"